### PR TITLE
Update gradle to 7.0

### PR DESCRIPTION
**Pull request by pashamcr**
Signed-off-by: pashamcr <36408873+pashamcr@users.noreply.github.com>

### CODE REVIEW

**Comment by Goooler** [comment]
👍

File: build.gradle [hunk 1]
```diff
@@ -127,7 +127,7 @@ subprojects {
             commons_lang: 'org.apache.commons:commons-lang3:3.12.0',
             guava: 'com.google.guava:guava:30.1.1-jre',
             junit: 'junit:junit:4.13.2',
-            proguard_gradle: 'com.guardsquare:proguard-gradle:7.0.0',
+            proguard_gradle: 'com.guardsquare:proguard-gradle:7.1.0-beta2',
```
Comment (iBotPeaches) [inline build.gradle:130]: I'd rather wait until a stable version. Don't want to risk betas and rushing to 7.

**Review by iBotPeaches — CHANGES_REQUESTED** [review]
I don't want to upgrade to any beta dependencies. I'm in no hurry. I can wait for proguard to stabilize with Gradle 7.

**Comment by Oizaro** [comment]
@iBotPeaches I think you can already merge this push by downgrading proguard_gradle to version 7.0.1 (last stable version).

I did it for my personal build and I had no problems.

**Comment by iBotPeaches** [comment]
> @iBotPeaches I think you can already merge this push by downgrading proguard_gradle to version 7.0.1 (last stable).
> 
> I did it for my personal build and I had no problems.

Thanks, I see beta 5 is already out for 7.1. I'm in no hurry, so I don't mind waiting for that to be stable. Apktool doesn't need to be on bleeding edge gradle/proguard tools and I heard gradle7 was a big jump that proguard needs to be compatible for.

**Comment by Goooler** [comment]
@pashamcr [proguard-gradle 7.1](https://github.com/Guardsquare/proguard/releases/tag/v7.1.0) is out, you can upgrade it to stable version. And the [gradle 7.1](https://docs.gradle.org/7.1/release-notes.html) should also be updated.

**Pull request merged**